fe/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -183,6 +183,13 @@
      * you can try to increase this value to decrease the chances of false timeouts
      */
     @ConfField public static int bdbje_heartbeat_timeout_second = 30;
+
+    /*
+     * The lock timeout of bdbje operation
+     * If there are many LockTimeoutException in FE WARN log, you can try to increase this value
+     */
+    @ConfField
+    public static int bdbje_lock_timeout_second = 1;
```

Comment:
Is 1 second enough for most cases?

Reply:
I‘m not sure. I internally use 3s always in Doris 0.9 and Doris 0.10 for a long time, and don't see `LockTimeoutException`.